hasura/migrations/1630635244186_create_table_public_PlayerRole/up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE "public"."PlayerRole"("role" text NOT NULL, "label" text NOT NULL, PRIMARY KEY ("role") , UNIQUE ("role"));
+
+INSERT INTO "public"."PlayerRole" (role, label) 
+VALUES ('PATRON', 'Patron'), ('BUILDER', 'Builder'), ('ARTIST', 'Artist'), ('SHILLER', 'Shiller'), ('INNKEEPER', 'Innkeeper'), ('DESIGNER', 'Designer'), ('WRITER', 'Writer');
```

Comment:
Any specific reason not all roles are in here?

As for the current guilds I'm missing:
Bughunter
Headhunter
XP-Fairy
Necromancer

As for other roles I'm missing:
Productizer
Rainmaker
Videolord
Soundmaster
Rogue
Bridgebuilder

Reply:
Yes, this is intentional and addressed in #311.